docs/accessibilityinfo.md

```diff
@@ -23,22 +23,22 @@ const App = () => {
   useEffect(() => {
     const reduceMotionChangedSubscription = AccessibilityInfo.addEventListener(
       'reduceMotionChanged',
-      reduceMotionEnabled => {
-        setReduceMotionEnabled(reduceMotionEnabled);
+      isReduceMotionEnabled => {
```

Comment:
A lot of these renamings throughout the PR are to avoid shadowing (so we can get clean of ESLint warnings)